app/src/main/java/com/infomaniak/mail/ui/main/thread/ThreadAdapter.kt

```diff
@@ -599,6 +587,7 @@ class ThreadAdapter(
         val fullMessageWebViewClient get() = _fullMessageWebViewClient!!
 
         init {
+            if (isForPrinting) WebView.enableSlowWholeDocumentDraw()
```

Comment:
Is this still needed now that you user the webview adapter and not the bitmap strategy?

Reply:
I just tried and it seems to work fine without. I'm removing this.